lib/pages/logged_in/run_page/map_and_location_logic/loading_map.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+class LoadingMap extends StatelessWidget {
+  const LoadingMap({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
Is there a reason this widget is in it's own file rather than with the run_page ?